src/Stepper/Step.tsx

```diff
@@ -38,9 +35,8 @@ const cssClass = {
 };
 
 const defaultProps = {
-  showTitle: true,
-  showMessage: true,
-  showDescription: true,
+  state: "INCOMPLETE",
+  warning: "We ran into an error on this step",
```

Comment:
i think we should just leave out this default. we should encourage clients to have more useful error messages than this